app/ui/src/Pages/Project/pages/KG/components/Filters/components/ScoreFilter/useScoreFilter.ts

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+const normalizer = 100;
+
+export interface ScoreFilterProps {
+  max?: number;
+  initialMin?: number;
+  initialMax?: number;
+}
+function useScoreFilter({
+  max = 100,
+  initialMin = 0,
+  initialMax = 100,
+}: ScoreFilterProps) {
+  const [scores, setScores] = useState<number[]>([initialMin, initialMax]);
```

Comment:
This typing is more appropiate:
```suggestion
  const [scores, setScores] = useState<[number, number]>([initialMin, initialMax]);
```